frontend/apps/reader/modules/readerzooming.lua

```diff
@@ -527,7 +527,7 @@ function ReaderZooming:getZoom(pageno)
         local ubbox_dimen = self.ui.document:getUsedBBoxDimensions(pageno, 1)
         -- if bbox is larger than the native page dimension render the full page
         -- See discussion in koreader/koreader#970.
-        if ubbox_dimen.w <= page_size.w and ubbox_dimen.h <= page_size.h then
+        if (ubbox_dimen.w <= page_size.w and ubbox_dimen.h <= page_size.h) or (self.ui.document.configurable.trim_page == 1) then
```

Comment:
I just meant the first one really, on the second it looks weird. :-P

But pinging @poire-z for some thoughts.

Reply:
I don't mind the parens around the second one. It looks balanced, and may save 500ms of brain processing power while reading :)